packages/build/src/plugins_core/edge_functions/index.js

```diff
@@ -69,6 +65,12 @@ const coreStep = async function ({
 
   await validateEdgeFunctionsManifest({ buildDir, constants: { EDGE_FUNCTIONS_DIST: distDirectory } })
 
+  if (!isRunningLocally) {
+    const logsLink = `https://app.netlify.com/sites/${env.SITE_NAME}/edge-handlers?scope=deployid:${env.DEPLOY_ID}`
```

Comment:
I think this is the right path?

```suggestion
    const logsLink = `https://app.netlify.com/sites/${env.SITE_NAME}/edge-functions?scope=deployid:${env.DEPLOY_ID}`
```